libs/model/src/policies/ContestBot.policy.ts

```diff
@@ -0,0 +1,80 @@
+import { ServerError } from '@hicommonwealth/core';
```

Comment:
This is a command that will be invoked by the bot policy.... let's rename it to something like CreateContest.command or something like that under the bot namespace